### ISSUE TITLE
[Feature Request] Disable DarkReader on specific page elements

### ISSUE THREAD

**Issue by LoganJFisher** [darkreader/darkreader#11313]
### Prerequisites

- [X] I [searched for any existing report](https://github.com/darkreader/darkreader/issues?q=is%3Aissue) about this feature request to avoid opening a duplicate.

### Is this feature request related to a problem?

No

### Feature Request Description

I would like the option to disable DarkReader not just on a given website, but for specific page elements on websites.

For example, Overleaf.com looks great with DarkReader enabled, but it shouldn't be applied to compile screen since seeing correct colors is important there.

Thank you.

**Comment by sherryysj** [comment]
Yeah, it will be great if it can be disabled for certain elements.

**Comment by MauricioPimenta** [comment]
Or maybe disable it for a subdomain, like **overleaf.com/project/\***...
Is there any way to do this?

**Comment by MauricioPimenta** [comment]
### [QUICK FIX]
If you add ".pdfViewer" to the INVERT tag on the Dynamic Theme Editor, it will show the pdf with the normal colors.

I thought about submitting a change to the dynamic-theme-fixes.config, but I don't think it would be accepted. In the [contributing.md](https://github.com/darkreader/darkreader/blob/main/CONTRIBUTING.md) it says:

> "In Dynamic mode, if the page looks partially dark and bright, it is considered a bug."

Because of this, I don't think it would be considered a bug with the dynamic theme, but you can add it locally in the developer tools of the extension:

- Open the darkreader extension menu and go to **Settings -> Advanced -> Dev tools** and search for the **overleaf.com** in the list of the **Dynamic Theme Editor**.
- Add **.pdfViewer** to the list of inversions.

It should look something like this:

```
[...]
================================

overleaf.com

INVERT
.canvasWrapper
.pdfViewer

================================
[...]
```

**Comment by ryanwwest** [comment]
I'd like this too. For Overleaf specifically, another issue is the vim cursor with dark mode enabled is very hard to see against the background, and the same occurs sometimes when using ctrl-f highlighting page results. While a default visible cursor and highlights are best, for situations where DarkReader misses the mark it would be great to cherrypick better colors.
